csrc/scheduler/reduction_utils.h

```diff
@@ -86,7 +86,8 @@ TORCH_CUDA_CU_API TensorView* sortAndRFactor(TensorView* reference_tv);
 // function create dummy outputs which should be used in later stages of the
 // scheduling.
 TORCH_CUDA_CU_API std::vector<TensorView*> projectPersistentBuffers(
-    Fusion* fusion);
+    Fusion* fusion,
+    const bool project_to_inputs = true);
```

Comment:
will do. I was trying to minmize the influence on the existing calls to this function. But since this function is rarelly directly used and we are not releasing yet, this backward compatability is not an issue.